src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/FrameOfT.cs

```diff
@@ -95,6 +96,8 @@ public Frame(IHttpApplication<TContext> application, FrameContext frameContext)
                         {
                             try
                             {
+                                KestrelEventSource.Log.RequestStart(this);
```

Comment:
Hosting only [logs method and path](https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Hosting/Internal/HostingEventSource.cs#L36-L39). This logs connectionId and requestId to make it possible to correlate connections and requests.